aws/resource_aws_default_vpc.go

```diff
@@ -28,6 +28,7 @@ func resourceAwsDefaultVpc() *schema.Resource {
 	// assign_generated_ipv6_cidr_block is a computed value for Default VPCs
 	dvpc.Schema["assign_generated_ipv6_cidr_block"] = &schema.Schema{
 		Type:     schema.TypeBool,
+		Optional: true,
```

Comment:
For backwards compatibility I don't set the `Default: false` flag here as is done for the equivalent attribute in the `aws_vpc` resource.